front/utils/CurrencyUtils.js

```diff
@@ -0,0 +1,26 @@
+import { get } from 'lodash'
+import Transaction from '~/models/Transaction'
+
+export function convertAmount(amount, fromCurrency, toCurrency, exchangeRates) {
+  const exchangeSource = get(exchangeRates, `rates.${fromCurrency}`)
+  const exchangeDestination = get(exchangeRates, `rates.${toCurrency}`)
+  return (1.0 * amount * exchangeDestination) / exchangeSource
+}
+
+export function transactionAmountInAccountCurrency (transaction, accountCurrency, exchangeRates) {
```

Comment:
Rename method to something simliar to the first function `convertAmount`, ex. `convertTransactionAmountToDashboardCurrency` and keep only the first param.